MLS.Agent.Tests/DotnetKernelJupyterInstallerTests.cs

```diff
@@ -15,20 +16,25 @@ public class DotnetKernelJupyterInstallerTests
         [Fact]
         public async Task Should_read_the_jupyter_paths_and_give_the_data_paths()
         {
+            var dataPath1 = @"C:\Users\AppData\Roaming\jupyter";
+            var dataPath2 = @"C:\Users\AppData\Local\Continuum\anaconda3\share\jupyter";
+            var dataPath3 = @"C:\ProgramData\jupyter";
+
             var pathsOutput = 
 $@"config:
-    C:\Users\akagarw\.jupyter
-    C:\Users\akagarw\AppData\Local\Continuum\anaconda3\etc\jupyter
-    C:\ProgramData\jupyter
+    C:\Users\.jupyter
 data:
-    C:\Users\akagarw\AppData\Roaming\jupyter
-    C:\Users\akagarw\AppData\Local\Continuum\anaconda3\share\jupyter
-    C:\ProgramData\jupyter
+   {dataPath1}
+   {dataPath2}
+   {dataPath3}
 runtime:
-    C:\Users\akagarw\AppData\Roaming\jupyter\runtime".Split("\n");
+    C:\Users\AppData\Roaming\jupyter\runtime".Split("\n");
 
             var dataDirectories = await DotnetKernelJupyterInstaller.GetJupyterDataPaths(pathsOutput);
             dataDirectories.Should().HaveCount(3);
+            dataDirectories.First().FullName.Should().Be(dataPath1);
+            dataDirectories.Should().Contain(dir => dir.FullName == dataPath1);
+            dataDirectories.Should().Contain(dir => dir.FullName == dataPath1);
```

Comment:
These three assertions look redundant.

You can do something like this:

```cs
dataDirectories.Should().BeEquivalentTo(dataPth1, dataPath2, dataPath3);
```

Reply:
Done. Thanks!